frontend/src/components/scheduleEntry/dateHelperLocal.js

```diff
@@ -1,26 +1,20 @@
-function defineHelpers (scheduleEntry, timed = false) {
+function defineHelpers (dayjs, scheduleEntry, timed = false) {
   if (!Object.prototype.hasOwnProperty.call(scheduleEntry, 'startTime')) {
     Object.defineProperties(scheduleEntry, {
       startTime: {
         get () {
-          // need to cut-off UTC Timezone such that the date is interpreted in local timezone setting (because v-calendar has no timezone setting)
-          const [periodStart] = this.period().start.split('+')
-
-          return Date.parse(periodStart) + (this.periodOffset * 60000)
+          return dayjs(this.period().start, dayjs.HTML5_FMT.DATE).add(this.periodOffset, 'm').valueOf()
```

Comment:
The actual problem was: `this.period().start` has the format `YYYY-MM-DD`, and calling `Date.parse` on that assumes UTC time, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#date_time_string_format.

> When the time zone offset is absent, date-only forms are interpreted as a UTC time and date-time forms are interpreted as local time.

It might have had a different format some time ago, otherwise I can't explain the useless timezone cutoff code. I refactored all the `dateHelper*` code using `dayjs`. Now you can see whether UTC or local is used: `dayjs.utc(...)` vs `dayjs(...)`.

I'm wondering whether we still want to use unix timestamps as the basic format in `startTime` and `endTime`. This causes a lot of parsing and formatting back and forth. Maybe the actual `dayjs` object would be a better fit?

Reply:
Maybe I misunderstand what you mean by this, but I think `startTime` and `endTime` are directly accessed by the Vuetify calendar. That's why we set `event-start="startTime"` in Picasso.vue. I doubt it would work it if we started handing out `dayjs` objects from these getters instead of UNIX timestamps.

There is an error message in the Vuetify [`parseTimestamp` function](https://github.com/vuetifyjs/vuetify/blob/7a361725a09e31f11bb6b414748a7ad55c0b3500/packages/vuetify/src/components/VCalendar/util/timestamp.ts#L114) that states that eventStart `must be a Date, number of seconds since Epoch, or a string in the format of YYYY-MM-DD or YYYY-MM-DD hh:mm. Zero-padding is optional and seconds are ignored.` The date string assumes local time according to the former comment in this file:
> the date is interpreted in local timezone setting (because v-calendar has no timezone setting)

Reply:
I will follow up with a proposal of what I had in mind here